backend/btrixcloud/db.py

```diff
@@ -141,6 +141,25 @@ async def run_db_migrations(mdb, user_manager):
     return migrations_run
 
 
+# ============================================================================
+async def await_db_and_migrations(mdb, db_inited):
+    """await that db is available and any migrations in progress finish"""
+    await ping_db(mdb)
+    print("Database setup started", flush=True)
+
+    base_migration = BaseMigration(mdb, CURR_DB_VERSION)
+    while await base_migration.migrate_up_needed():
+        version = await base_migration.get_db_version()
+        print(
+            f"Waiting for migrations to finish, DB at {version}, latest {CURR_DB_VERSION}"
+        )
```

Comment:
```suggestion
        print("Waiting for migrations to finish in other worker", flush=True)
```

^ To avoid confusing users if the version in this worker is behind the one that's actually doing the migrations.

Reply:
Resolved in latest commit by adding `flush=True` to print statement so that it doesn't wait for the buffer to fill.

Reply:
Was logging too late, fixed with flush=True